lang/RStudio/README.md

```diff
@@ -0,0 +1,46 @@
+# RStudio
+
+RStudio is an Integrated Development Environment (IDE) for the R language. By containerizing RStudio, a scientist can leverage the reproducibility and portability of the Singularity platform, allowing them to build data visualizations and analysis tools and ensure that they behave exactly the same in a new environment. 
+
+The definiton file affords some flexibility, allowing you to run RStudio several different ways:
+
+Instance:
+ This will start up rstudio-server, on default port of 8787
+
+$ singularity instance start rstudio.sif rs
+Run:
+ This will run rstudio IDE
+
+$ singularity run rstudio.sif
+Exec:
+ Any program in the container, but specifically for running R directly.
+
+$ singularity exec rstudio.sif R -f myfile.r
+To build the image run:
+
+$ sudo singularity build rstudio.sif rstudio.def
+When you use run for the IDE, you may need to execute it like:
```

Comment:
Of course, again the formatting of markdown for blocks of code so between the triple quote mark.. ```